src/components/item/form/AppForm.tsx

```diff
@@ -147,19 +163,35 @@ const AppForm = ({ onChange, updatedProperties = {} }: Props): JSX.Element => {
           />
         </Stack>
       ) : (
-        <Grid2 container spacing={2} alignItems="stretch" overflow="scroll">
-          <AppGrid
-            currentUrl={currentUrl}
-            handleSelection={handleAppSelection}
-          />
-          <AppCard
-            name={translateBuilder(BUILDER.CREATE_CUSTOM_APP)}
-            description={translateBuilder(
-              BUILDER.CREATE_CUSTOM_APP_DESCRIPTION,
-            )}
-            onClick={addCustomApp}
+        <>
+          <TextField
+            fullWidth
+            placeholder={translateBuilder('Search for an app')}
+            variant="outlined"
+            size="small"
+            onChange={searchAnApp}
           />
-        </Grid2>
+          <Grid2
+            container
+            spacing={2}
+            maxHeight={400}
```

Comment:
The modal will have a maximum height and the scroll will adapt, but we see a big height difference between "app" and "link" for example when you switch, I don't really like it, especially because if moves a little bit the menu on the right. Let me know what you think.